bridge/docs/bridging.md

```diff
@@ -34,3 +34,91 @@ Again, a withdrawfee of 1 TFT will be taken, so make sure you send a larger amou
+4. `tftBridgeModule.RefundTransactionProcessed`: A bridge validator has called `set_refund_transaction_executed` extrinsic with a proof that the proposed stellar refund transaction was executed successfully on stellar network.
+
+### TFCHAIN -> STELLAR (Burn-and-Withdraw flow)
+now, we look into the details of transferring TFT from a TFChain Account to a Stellar.
+
+1. To withdraw your asset back to Stellar, the TFTBridgeModule's `swap_to_stellar` extrinsic in TFChain must be called with an amount to burn (on the TFChain side) and a Stellar account ID to receive the equivalent TFT amount (on the Stellar network side).
+
+2. The call validates the target Stellar account ID and ensures that you have enough balance in the source account. If so, it burns the amount and transfer fees to feeAccount, increments the `BurnTransactionId` in the TFTBridgeModule storage, store data about the transaction with empty signatures placeholder, adds it to `BurnTransactions` with the `burnId` as key, and emits `BurnTransactionCreated` event. This event contains `burn_id`, `source` account, `target` Stellar address, and burn `amount`.
+
+3. The bridge validators are listening for this event. They extract the `burnId` and other transaction parameters, validate the Stellar address (tokens could refunded/minted back on TFChain at this step if validation failed), then construct signed stellar transaction and extract the signature (note, the transaction can not be submitted yet to stellar network). They then call `propose_burn_transaction_or_add_sig` extrinsic which fill their signatures and the bridge account sequence number in the `BurnTransaction` in storage that matches specified `burnId`. when this call executed, the `BurnTransactionSignatureAdded` event is emitted. 
+
+4. If the majority (more than the half) of bridge validators provided their signature for a transaction, a `BurnTransactionReady` event is emitted as well. This check happens every time the `propose_burn_transaction_or_add_sig` extrinsic is executed by validator call.
+
+5. The bridge will handle the event and query TFChain storage for the `BurnTransaction` details and the validators’ signatures. It will create a multi-signatures Stellar transaction and submit it to Stellar network. If submitted successfully, it will call `set_burn_transaction_executed` extrinsic (which removes the `BurnTransaction` from the `BurnTransactions` storage and adds it to `ExecutedBurnTransactions`) then emit `BurnTransactionProcessed` event.
+
+#### Overview of the TFChain Burning events
+1. `tftBridgeModule.BurnTransactionCreated`: A swap from TFChain to stellar was initiated by a call to `swap_to_stellar` extrinsic.
+2. `tftBridgeModule.BurnTransactionSignatureAdded`: Other bridge validators handled `BurnTransactionCreated` TFChain event and submit their signature for the proposed stellar transaction.
```

Comment:
A bridge validator handled `BurnTransactionCreated` TFChain event and submited its signature for the proposed stellar transaction.